azure-pipelines.yml

```diff
@@ -30,6 +30,11 @@ extends:
         timeoutInMinutes: 360
 
         variables:
+        - name: deploymentSlot
+          value: 'validation'
+        - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/main')) }}:
```

Comment:
IIUC staging is the production slot, and will only be used for internal non-PR builds of main?

Reply:
The workflow is: builds from main deploy to `staging`, and are promoted to `prod` if they pass CI. in Theory, prod should never be broken by a bad build.